src/quo2/components/settings/settings_list/style.cljs

```diff
@@ -0,0 +1,63 @@
+(ns quo2.components.settings.settings-list.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.platform :as platform]))
+
+(def title
+  {:text-color (colors/theme-colors
+                colors/neutral-100
+                colors/white)
+  })
+
+(def icon
+  {:margin-right 12})
+
+(def item-container
+  {:justify-content    :space-between
+   :margin-bottom      24
+   :height             48
+   :flex-direction     :row
+   :align-items        :center
+   :padding-horizontal 12
+   :padding-vertical   13
+   :margin-horizontal  0
+   :margin-vertical    8})
+
+(def switch-button
+  (if platform/ios?
+    {:transform [{:scaleY 0.588} {:scaleX 0.645}]}
+    {:transform [{:scaleY 0.625} {:scaleX 0.416}]}))
+
+(def dot
```

Comment:
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#styles-def-vs-defn

You should use `defn` here. Otherwise, the background color won't get updated when we change the theme.